packages/thirdweb/src/extensions/lens/read/resolveAddress.ts

```diff
@@ -0,0 +1,80 @@
+import type { Chain } from "../../../chains/types.js";
+import type { ThirdwebClient } from "../../../client/client.js";
+import { getAddress, isAddress } from "../../../utils/address.js";
+
+/**
+ * @extension LENS
+ */
+export type ResolveLensAddressParams = {
+  handleOrLocalName: string;
+  client: ThirdwebClient;
+  overrides?: {
+    lensHandleAddress?: string;
+    chain?: Chain;
+  };
+};
+
+/**
+ * Take in a Lens handle or local-name and return the wallet address behind that handle/local-name.
+ * For example, "lens/vitalik" is a handle, with "lens" being the namespace and "vitalik" being the local name
+ * @extension LENS
+ * @example
+ * ```ts
+ * import { resolveAddress } from "thirdweb/extensions/lens";
+ *
+ * const walletAddress = await resolveAddress("vitalik");
```

Comment:
example is wrong right? should be an object

Reply:
oops I'll fix